wallet/txrules/rules.go

```diff
@@ -39,14 +39,14 @@ var (
 
 // CheckOutput performs simple consensus and policy tests on a transaction
 // output.
-func CheckOutput(output *wire.TxOut, relayFeePerKb btcutil.Amount) error {
+func CheckOutput(output *wire.TxOut, relayFeePerKb btcutil.Amount, allowDust bool) error {
```

Comment:
Since this is a breaking API change, should we just create a new function `CheckOutputAllowDust` that has the boolean flag? Then this function can call into the new one with `false`.